test/BaselineOutput/Common/LdSvm/netcoreapp/LDSVM-def-CV-breast-cancer-out.txt

```diff
@@ -11,43 +11,43 @@ Confusion table
           ||======================
 PREDICTED || positive | negative | Recall
 TRUTH     ||======================
- positive ||      134 |        0 | 1.0000
- negative ||       12 |      208 | 0.9455
+ positive ||      133 |        1 | 0.9925
+ negative ||       10 |      210 | 0.9545
           ||======================
-Precision ||   0.9178 |   1.0000 |
-OVERALL 0/1 ACCURACY: 0.966102
-LOG LOSS/instance:  0.121887
+Precision ||   0.9301 |   0.9953 |
+OVERALL 0/1 ACCURACY: 0.968927
```

Comment:
Looks like this is slightly higher accuracy, so a net improvement?

Reply:
The biggest thing is just that the dataset we are using for these tests is tiny, so any minor change we make to anything will affect the accuracy. The tests I ran on actual full size datasets didn't really show any changes in accuracy.